src/SFA.DAS.AANHub.Api/Common/BaseRequestDetails.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace SFA.DAS.AANHub.Api.Common
+{
+    public class BaseRequestDetails
```

Comment:
RefererRouteDetails

Reply:
Class name changed to `ReferrerRouteDetails`